source/guides/first-steps/task-script.md

```diff
@@ -1,4 +1,121 @@
+
+Once your file has been created, you can name it something like `MyTask.dsc`. Make sure to use the `.dsc` extension so VS Code recognizes it as a Denizen script, but the *name* of the file can be whatever you like. Usually, it's helpful to name the file something relevant to the script(s) that it contains.
+
+From here, double-click the file to open it in VS Code.
+
+TODO SCREENSHOT (opened file)
+
+Now you can begin writing your first task script!
+
+#### Writing the script
+
+Let's start with the core of the script:
+
+```dscript_blue
+my_first_task:
+    type: task
+    script:
+    - narrate (sometext)
+```
+
+This example should look familiar - it's very similar to the example above.
+
+### Script names
+
+The name of the script here is `myfirsttask`. It's at the top of the indentation. Each top-level indentation entry is a separate script. Here is an example of two different scripts:
+
+```dscript_green
+myfirsttask:
+    type: task
+    script:
+    - narrate "This is is task number one!"
+
+mysecondtask:
+    type: task
+    script:
+    - narratee "This is task number two!"
+```
+
+This example demonstrates what having two different scripts in the same file looks like. `myfirsttask` and `mysecondtask` are two different scripts, and each is a fully self-contained script.
+
+### Script types
+
+Below the script name - on the second level of indentation - you'll see the `type` and `script` entries.
+
+The `type` option is where you specify which `type` of Denizen script you're writing. In this case, we're writing a `task` script. You may have seen `world` scripts, `item` scripts, `inventory` scripts, and *maybe* even a `procedure` script. For now, let's focus on `task` scripts. Make sure you write `type: task` in your script on the second level of indentation, below the script name, like this:
+
+```dscript_blue
+myfirsttask:
+    type: task
+```
+
+### Script Commands
+
+Below the `type` entry, you'll see the `script` entry. This is where you'll be doing most of your work in Denizen! Under the `script` entry is where you write the content of the script - no matter which kind of script it is, the `script` entry is where you tell Denizen what to do. There are many different script types - a few of them being world scripts, assignment scripts, command scripts, and item scripts. You'll learn about the different types as you read through these guides, and you can find explanations about all of them from the meta documentation on the [language explanation page](https://one.denizenscript.com/denizen/lngs/).
+
+Let's look at another example script:
+
+```dscript_green
+my_first_task:
+    type: task
+    script:
+    - narrate "This is a valid task script!"
+```
+
+This script only has one command in it, the `narrate` command. As a reminder, `narrate` will display the specified text to the attached player. You can also specify a custom list of targets to `narrate` to if you want to `narrate` to multiple players!
```

Comment:
This line doesn't belong on this page. Somebody who's reading this page as a newbie wouldn't yet have any idea how to do something so complex as handling a list of players.

Reply:
You mention `This script only has one command in it`, but that inspires something that probably should be in here that isn't yet: a sample script with multiple lines! Nothing fancy, just two or three narrates in a row, just to show how that would look.
Maybe even a task script that narrates a few things, then runs a second task script that also narrates, just to show a different command in use, and as well show how you run a task script from inside a script.